tests/README.md

```diff
@@ -0,0 +1,12 @@
+# Testing
+
+*Unittest is the preferred test framework for this project*
+
+## Run all tests
+
+From the top directory:
+
+```
+python -m unittest discover -vv -s tests -t $PWD
+```
+
```

Comment:
Please do a re-base with master and move your test directories under `tests/unit/`.  Also, add copyright header to all `__init__.py` files